proposals/vector_reproject.json

```diff
@@ -0,0 +1,97 @@
+{
+    "id": "vector_reproject",
+    "summary": "Reprojects the geometry dimension",
+    "description": "Converts the geometries stored in a geometry dimensions to a different projection.\n\nTo spatially align two data cubes with each other, better use the process ``resample_cube_spatial()``.",
```

Comment:
```suggestion
    "description": "Converts the geometries stored in a geometry dimensions to a different projection.\n\nTo spatially align two data cubes with each other, better use the process `resample_cube_spatial()`.",
```

Reply:
The two characters are correct as it enables the internal linking between processes.

Reply:
Ah, apologies, thanks for clarifying! :)